packages/ra-supabase-core/README.md

```diff
@@ -49,6 +49,22 @@ export const authProvider = supabaseAuthProvider(supabase, {
             fullName: `${data.first_name} ${data.last_name}`,
         };
     },
+    getPermissions: async user => {
```

Comment:
**polish**: Following the example of the `getIdentity` paragraph, would you mind to mention and explain this method in the **API -> The `supabaseAuthProvider`** section?